calc.py

```diff
@@ -39,9 +39,16 @@ def add(*args):
 
 def log10(x):
 	"""Return a base-10 logarithm of x"""
-	
+
```

Comment:
It is good practice to remove white space or empty lines between the doc string and the code